tests/queries/0_stateless/01052_window_view_proc_tumble_to_now.sql

```diff
@@ -2,17 +2,17 @@ SET allow_experimental_window_view = 1;
 
 DROP TABLE IF EXISTS mt;
 DROP TABLE IF EXISTS dst;
-DROP TABLE IF EXISTS wv;
+DROP TABLE IF EXISTS wv NO DELAY;
```

Comment:
AFAIK it has no effect, because `database_atomic_wait_for_drop_and_detach_synchronously` settings is enabled in CI

Reply:
Ok, but it is not the fix for flackyness. It is for convenience to run tests locally.

Reply:
Hm, but it can work even with disabled setting and without `NO DELAY/SYNC` modifier. At least, MaterializedView can:
https://github.com/ClickHouse/ClickHouse/blob/f0ee0724ace333206005ef0d65f5389b44688ba8/src/Storages/StorageMaterializedView.cpp#L39-L44

Reply:
Oh, good point! I need to add the same check
```
if (view_id.hasUUID()) 
         return ".inner_id." + toString(view_id.uuid); 
```
for window view.
Then there will be no inconvenience.